gapis/gfxapi/vulkan/vulkan.api

```diff
@@ -2766,6 +2766,7 @@ cmd void vkGetDeviceQueue(
   pQueue[0] = id
 }
 
+@DrawCall
```

Comment:
Is every vkQueueSubmit guaranteed to contain a draw call? Isn't it also possible to have multiple draw calls in a command buffer? If either is true, please add a TODO (somewhere) to indicate the current limitations.

With the proto changes we're currently making, it might be helpful to give Ben some info here: https://github.com/ben-clayton/gapid/issues/17